Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -144,6 +146,8 @@ class AppSettingsTableViewController: SettingsTableViewController {
             return 64
         } else if section[indexPath.row] is ConnectSetting {
             return 64
+        } else if let setting = section[indexPath.row] as? BoolSetting where setting.prefKey == "settings.allowThirdPartyKeyboards" {
```

Comment:
Nit: I think at this point we can collapse all of these else ifs into a single statement since they all return the same thing (64). Down the road it might be nice to add a height property to Settings so we can throw away the if check all together and just return the setting's height.